public/notify.min.js

```diff
@@ -1 +1,296 @@
\ No newline at end of file
+(function(e) {
```

Comment:
No need for these changes, min files are supposed to be like this. Remove changes from this file.